tests/suit/framework/ovirt.go

```diff
@@ -49,11 +59,10 @@ func (r *OvirtClient) LoadSourceDetails() (vm *OvirtVM, err error) {
 	// default test values
 	sdomains := []string{"95ef6fee-5773-46a2-9340-a636958a96b8"}
 	nics := []string{"6b6b7239-5ea1-4f08-a76e-be150ab8eb89"}
-
 	// if a real custom ovirt environment is used.
 	if r.CustomEnv {
 		defer r.Close()
-		// connect to ovirt instance
+		// connect/insecure connect to ovirt instance
```

Comment:
I'd change this to
```suggestion
		// connect (possibly insecurely) to ovirt instance
```
but is it really important to mention this??

Reply:
nope, removing:)